x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -48,5 +90,127 @@ describe('Resolver Data Middleware', () => {
     it('should indicate there are additional children', () => {
       expect(selectors.hasMoreChildren(store.getState())).toBe(true);
     });
+    it('should have the correct related events', () => {
+      const selectedEventsByEntityId = selectors.relatedEventsByEntityId(store.getState());
+      const selectedEventsForFirstChildNode = selectedEventsByEntityId.get(firstChildNodeInTree.id)!
```

Comment:
why would you pass `.id` instead of the entity ID?

Reply:
`TreeNode`s don't have entityIDs